nox/_option_set.py

```diff
@@ -242,8 +240,8 @@ def parser(self) -> ArgumentParser:
             argument = groups[option.group.name].add_argument(
                 *option.flags, help=option.help, default=option.default, **option.kwargs
             )
-            if getattr(option, "completer"):
-                setattr(argument, "completer", option.completer)
+            if option.completer:
```

Comment:
Also not safer/different than the default syntax. Was this supposed to be hasattr, perhaps?

Reply:
Agreed 👍🏻